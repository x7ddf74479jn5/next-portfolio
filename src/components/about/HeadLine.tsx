import styles from "src/styles/components/about/Headline.module.scss";

const HeadLine: React.FC = ({ children }) => {
  return <h2 className={styles.headlineAbout}>{children}</h2>;
};

export default HeadLine;
