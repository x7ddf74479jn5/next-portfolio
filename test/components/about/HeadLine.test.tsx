import { render } from "@testing-library/react";
import HeadLine from "src/components/about/HeadLine";

describe("HeadLine", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<HeadLine />);
    expect(asFragment()).toMatchSnapshot();
  });
});
