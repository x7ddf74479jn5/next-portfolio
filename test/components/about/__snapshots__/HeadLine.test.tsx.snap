// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadLine matches snapshot 1`] = `
<DocumentFragment>
  <h2
    class="headlineAbout"
  />
</DocumentFragment>
`;
